docs/src/development/proposals/MEP6/README.md

```diff
@@ -0,0 +1,63 @@
+# DMZ Networks
+
+## Reasoning
+
+To fulfill higher levels of security measures the standard metal-stack approach with a single firewall in front of a set of machines might be insufficient.
+There are cases where two physically distinct firewalls in front of application workload are mandatory. In traditional network terms this is known as DMZ approach.
+
+For Kubernetes workloads it makes sense to use the front cluster for ingress, WAF purposes and as outgoing proxy. The clusters may be used for application workload.
+
+## Approach
+
+![DMZ Internet](dmz-internet.png)
+
+### DMZ network
+
+- Use a separate DMZ network prefix for every tenant
+- This is used as intermediate network btw. private networks of a tenant and the internet
+- For every partition a distinct DMZ firewall/cluster is needed for a tenant
+- For Gardener orchestrated Kubernetes clusters this network must be a publicly reachable internet prefix because shoot clusters need a vpn service that is used for instrumentation from the seed cluster - this will be a requirement as long as the inverse vpn tunnel feature Konnectivity is not available to us.
```

Comment:
konnectivity-tunnel is currently tested in test-environment, seems to work just fine.

shoots needs annoation:

```yaml
  alpha.featuregates.shoot.gardener.cloud/konnectivity-tunnel: "true"
```
and an additional cwnp:
```yaml
apiVersion: metal-stack.io/v1
kind: ClusterwideNetworkPolicy
metadata:
  name: allow-to-konnectivity
  namespace: firewall
spec:
  egress:
  - ports:
    - port: 8132
      protocol: TCP
    to:
    - cidr: 0.0.0.0/0 # better: make this the ip of the apiserver
   ```

Reply:
This would make things a lot easier to implement and to understand.